lib/config_validator.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var Joi = require('joi');
+
+var mapObj = require('./create_table_map.js').mapObj;
+
+// non empty, alphanumeric, no leading number, less than 64
+var dbNameRegEx = /^[A-Za-z_]\w{0,62}$/;
+var fieldTypes = Object.keys(mapObj);
+
+var fieldSchema = Joi.object()
+  .keys({ type: Joi.any().valid(fieldTypes) })
+  .unknown()
+;
+var configSchema = Joi.object().keys({
+  table_name: Joi.string().regex(dbNameRegEx).required(), // eslint-disable-line
```

Comment:
Ah, lame. although for `git diff` purposes splitting over more lines is more review-able... (_fine for now_)